src/CentOps.Api/appsettings.json

```
@@ -14,5 +14,8 @@
   },
   "ConnectionStrings": {
     "AdminApiKey": ""
+  },
+  "Cors": {
+    "AllowedOrigins":  ["*"]
```

I think this may have been missed. Could you change the `*` to `locahost` please?
```suggestion
    "AllowedOrigins":  ["localhost"]
```